test/installer.js

```diff
@@ -171,10 +171,8 @@ describe('module', function () {
         child.exec('dpkg-deb -x bartest_amd64.deb .', { cwd: dest }, function (err, stdout, stderr) {
           if (err) return done(err)
           if (stderr) return done(new Error(stderr.toString()))
-
           fs.readFile(dest + 'usr/share/applications/bartest.desktop', function (err, data) {
             if (err) return done(err)
-
```

Comment:
You shouldn't delete this or any of the empty lines. They are here to improve readability